### PR TITLE
Add missing % to format strings

### PR DESCRIPTION
**Pull request by mbolivar**
A few strings around the code base are not being properly formatted with %. Fix them.

Please note: I've accepted the contributor agreement under username https://os.mbed.com/users/martifoundries/.

### CODE REVIEW

File: pyocd/tools/pyocd.py [hunk 1]
```diff
@@ -896,10 +896,10 @@ def do_write(self, args, width):
             # Look up flash region.
             region = self.session.target.memory_map.get_region_for_address(addr)
             if not region:
-                print("address 0x%08x is not within a memory region", pagaddre_addr)
+                print("address 0x%08x is not within a memory region" % pagaddre_addr)
```
Comment (flit) [inline pyocd/tools/pyocd.py]: Since you're already touching on this line, would you mind fixing the bad `pagaddre_addr` variable name (I just noticed it)?

**Comment by flit** [comment]
Thanks for the fixes!

File: pyocd/tools/pyocd.py (hunk 1 above)
Comment (mbolivar) [reply pyocd/tools/pyocd.py]: Done!

Comment (flit) [reply pyocd/tools/pyocd.py]: Thanks!

**Pull request merged**